mesa/datacollection.py

```diff
@@ -177,6 +177,7 @@ def collect(self, model):
                 # Check if function with arguments
                 elif isinstance(reporter, list):
                     self.model_vars[var].append(reporter[0](*reporter[1]))
+                # Check if method of a class
```

Comment:
You meant to add "TODO" here? There doesn't seem to be any check if reporter is at least a callable.

Reply:
I also changed a bit the docstring